src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,12 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball firstBall = lottery.getRandomBall();
+        System.out.println(firstBall);
+        Ball secondBall = lottery.getRandomBall();
+        System.out.println(secondBall);
+        Ball thirdBall = lottery.getRandomBall();
+        System.out.println(thirdBall);
```

Comment:
Instead of creating each `Ball` object separately, consider using a loop to create and print the balls. This approach will make your code cleaner and more maintainable. Refer to the checklist for an example of how to use a loop for creating multiple objects .